src/MyMind.Web/Controllers/TestController.cs

```diff
@@ -14,8 +14,22 @@ public class TestController : BaseApiController
         private readonly ISessionFactory _sessionFactory;
 
         public TestController(ISessionFactory sessionFactory)
+            => this._sessionFactory = sessionFactory;
```

Comment:
nawet jak kod mieści się w jednej linii, to zazwyczaj lepiej wrzucać go w klamry, tak jest czytelniej

Reply:
ok